mne/annotations.py

```diff
@@ -58,7 +59,48 @@
 _datetime = datetime
 
 
-def _check_o_d_s_c(onset, duration, description, ch_names):
+class _DetailsDict(dict):
```

Comment:
I am not a fan of the word "details". It suggests it's not important. To me it's more extra fields.